Assets/Scenes/UI.prefab

```diff
@@ -0,0 +1,451 @@
+%YAML 1.1
```

Comment:
Can you explain what this prefab is for? And see [this discussion](https://cvmlp.slack.com/archives/C05HWV5UZ1A/p1707154210737549?thread_ts=1707148506.826349&cid=C05HWV5UZ1A). I wonder if we can convert this to a piece of readable code.

Reply:
This contains the UI overlay, shared by both scenes.

Reply:
Same as above, I'd strongly prefer that we create dynamically via script. We're setting a bad example for how to do reusability/readability if we do it this way.

Reply:
Ok, I took a closer look and I see that your `UI` prefab is more akin to data. It doesn't include any custom scripting. So maybe this is fine. Maybe you could rename it to better convey what it is? I think what I'm struggling with is that there's no way to document or inspect the "interface" of a prefab... what is it providing? The only way to understand is to literally open it up in the editor and browse its internals.